<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoShotty</class>
 <widget class="QMainWindow" name="AutoShotty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>254</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>140</width>
    <height>50</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>311</width>
    <height>254</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoShotty</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>123</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="compressButton">
       <property name="maximumSize">
        <size>
         <width>37</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string> ⇱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="expandButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>⇲</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>⏵</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>290</width>
      <height>174</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QPushButton" name="setBottomRightButton">
       <property name="text">
        <string>set bottom right</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="setNextPageButton">
       <property name="text">
        <string>set target app</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="filenameLineEdit">
       <property name="text">
        <string>output</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Quality</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="setTopLeftButton">
       <property name="text">
        <string>set top left</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="saveToButton">
       <property name="text">
        <string>save to...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filetype</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSlider" name="qualitySlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Pages</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Delay</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="delaySpinBox">
       <property name="suffix">
        <string> s</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>200.000000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="pagesSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="filetypeComboBox">
       <item>
        <property name="text">
         <string>PDF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pngs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jpgs</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="keyboardModeCheckbox">
       <property name="text">
        <string>keyboard mode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QComboBox" name="keyboardComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>→</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>←</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>enter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>page ↓</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>page ↑</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
